modelling/src/test/java/org/axonframework/modelling/saga/repository/legacy_jpa/CustomSerializedSaga.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.axonframework.modelling.saga.repository.jpa;
+package org.axonframework.modelling.saga.repository.legacy_jpa;
 
 import org.axonframework.serialization.SimpleSerializedObject;
```

Comment:
No tests for Jakarta for Sagas as well

Reply:
Added module with tests run against hibernate 6, using the jakarta classes.